### ISSUE TITLE
Connection pool support

### ISSUE THREAD

**Issue by jaumard** [schultek/stormberry#5]
Hey, I'm checking for a Postgres ORM and found your package, really like your approach. 
The query capabilities are quite light for now, having something like `users..where.id.equals(10)..where.name.like('%Jim')` a bit like what angel3_orm is doing is quite nice imho.

I don't see any support for connection pool, is it something you will consider? Would you accept pull request?

**Comment by schultek** [comment]
Yes I'm happy about any contribution. Feel free to open a pull request.

**Comment by schultek** [comment]
Connection pooling is supported since update to postgres 3.0.0
